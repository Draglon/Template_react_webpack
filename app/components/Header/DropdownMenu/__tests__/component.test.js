import React from 'react';

import DropdownMenuComponent from '../component';

describe('<DropdownMenuComponent />', () => {
  let component;
  const props = {
    deleteSessionRequest: () => {},
  };

  it('Render snapshot - DropdownMenuComponent', () => {
    component = shallow(<DropdownMenuComponent {...props} />);
    expect(component).toMatchSnapshot();
  });
});
