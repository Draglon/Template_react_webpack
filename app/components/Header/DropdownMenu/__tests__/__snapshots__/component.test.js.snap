// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DropdownMenuComponent /> Render snapshot - DropdownMenuComponent 1`] = `
<Menu>
  <MenuItem>
    <Link
      replace={false}
      to="/"
    >
      Dashboard
    </Link>
  </MenuItem>
  <Divider
    className=""
    disabled={true}
    style={Object {}}
  />
  <MenuItem>
    <Link
      replace={false}
      to="/mylists"
    >
      My Lists
    </Link>
  </MenuItem>
  <MenuItem>
    <Link
      replace={false}
      to="/watchlist"
    >
      Watchlist
    </Link>
  </MenuItem>
  <MenuItem>
    <Link
      replace={false}
      to="/favorites"
    >
      Favorites
    </Link>
  </MenuItem>
  <Divider
    className=""
    disabled={true}
    style={Object {}}
  />
  <MenuItem
    onClick={[Function]}
  >
    Logout
  </MenuItem>
</Menu>
`;
